src/main/python/keyboard_to_nt.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 FRC 5990
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+#lets go trigon
+
+#if this library is not installed, dont install ntcore but pyntcore
+import ntcore
+
+import keyboard
+import time
+
+team = 4590#GREENBLITZ 🐐🐐🐐🐐
+
+
+def main():
+    def on_action(event: keyboard.KeyboardEvent):
+        if event.name == "/" or event == None or event.name == None:
```

Comment:
why if event == "/", if its button think if you want to stop the program like this and if yes move to const

Reply:
because you cant put a slash in the network tables.
i cant put "SmartDashboard/Keyboard//" because that doesn't make sense.
i can put "SmartDashboard/Keyboard/." because . is not the seperator character

Reply:
i changed the condition. now i can dashboard it